repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -121,6 +124,7 @@
                     <xs:element name="artifact-directory" type="papic:ArtifactDirectory" minOccurs="1" maxOccurs="1"/>
                     <xs:element name="ssl-configuration" type="papic:SslConfiguration" minOccurs="0" maxOccurs="1"/>
                     <xs:element name="logging-configuration" type="papic:LoggingConfiguration" minOccurs="0" maxOccurs="1"/>
+                    <xs:element name="via-configuration" type="papic:ViaConfiguration" minOccurs="0" maxOccurs="1"/>
```

Comment:
I'm curious why you went with the name `via-configuration`.  I would have expected something like `via-header` which would be consistent with the `tracing-header` element in the System Model config.

As an aside, this schema is wildly inconsistent.  Sometimes it uses `config` while other times it spells out `configuration`.  Sometimes `configuration` simply implies a configuration file, whereas other times it's a section of configuration.  Sometimes the names are camel case but most times they are dash-separated.  But I digress.